src/Tabs.php

```diff
@@ -50,7 +50,7 @@ public function addTab($name = null, $action = null)
             } else {
                 // otherwise treat it as URL
                 //# TODO: refactor this ugly hack
-                $item->setPath(str_replace('.php.', '.', $this->app->url($action)).'#');
+                $item->setPath(str_replace('.php.', '.', $this->url($action)).'#');
```

Comment:
need to test this change

Reply:
did a manual test by adding sticky on form.php / form2.php, worked fine.